src/rotator_library/providers/qwen_code_provider.py

```
@@ -602,16 +607,23 @@ async def make_request():
             # Build clean payload with only supported parameters
             payload = self._build_request_payload(**kwargs_with_stripped_model)
 
+            try:
+                is_oauth = credential_path.startswith("env://") or os.path.isfile(credential_path)
```

Ruff (0.15.9)

[warning] Async functions should not use os.path methods, use trio.Path or anyio.path

(ASYNC240)